swf/src/avm1/read.rs

```diff
@@ -68,7 +66,7 @@ impl<'a> Reader<'a> {
     pub fn read_opcode_and_length(&mut self) -> Result<(u8, usize)> {
         let opcode = self.read_u8()?;
         let length = if opcode >= 0x80 {
-            self.read_u16()? as usize
+            self.read_u16()?.into()
```

Comment:
Side note: I generally dislike `.into()`, it makes stuff much more implicit in terms of typing (sometimes I need to jump back and forth a lot through the code to find what it's even being converted into) and complexity (whether the conversion is trivial or not). Especially in SWF parsing code, we had a lot of gotchas related to treating numeric types, so I felt like keeping casts explicit helps.

(But that's just my opinion)